lib/rouge/guessers/disambiguation.rb

```diff
@@ -140,12 +140,18 @@ def match?(filename)
 
         Puppet
       end
-      
+
       disambiguate '*.p' do
         next Prolog if contains?(':-')
         next Prolog if matches?(/\A\w+(\(\w+\,\s*\w+\))*\./)
         next OpenEdge
       end
+
+      disambiguate '*.st' do
+          next IecST if  contains?('END_')
+          next IecST if  contains?('(*')
```

Comment:
Is this always appearing at the start of the line? If it is we can make this regex a bit stricter to avoid false positives.

Reply:
Normally it's written at the beginning of a line, yes.
So something like `/^\s*END_/i` should be a good test.